time timestamp
float32 x
float32 y
float32 horizontalAccuracy
bool dataOk
